maintenance/measure_tile_performance/measure_tile_performance.py

```diff
@@ -128,26 +153,31 @@ def main():
 
         i += 1
 
+    # print the filename on console
     if args.file:
         print("Filename =", filename)
         f.close()
         
     #Print for Console 
     print("RunTimes =", query_list)
-    minimum = min(query_list)
-    print("Min =", minimum)
-    maximum = max(query_list)
-    print("Max =", maximum)
+    #average run time
     sum_of_list = sum(query_list)
     length = len(query_list)
     average = round(sum_of_list/length, 2)
     print("Avg =", average)
+    #max run time
+    maximum = max(query_list)
+    print("Max =", maximum)
+    #min run time
+    minimum = min(query_list)
+    print("Min =", minimum)
     #if query runs are more than 1, then calculate standard deviation
     if times > 1:
         standard_dev = round(statistics.stdev(query_list), 2)
         print("Standard Deviation =", standard_dev)
-    #close redshift connection
-    conn.close()
```

Comment:
You removed the connection close here, but you don't have close implemented as part of your `cache_on()` or `cache_off()` functions, which means any connections created here are retained in memory.

You should be closing connections once you're done using them.

Reply:
Like i explained above connection is being closed automatically using with statements in both functions.

Reply:
@veenupunyani Please review my comment on that matter above.

Reply:
Right, i have closed the connection in the latest commit